go.mod

```diff
@@ -3,7 +3,8 @@ module github.com/GetStream/stream-chat-go/v2
 go 1.12
 
 require (
```

Comment:
also time to bump this

Reply:
Do you mean `go 1.12`? If so I'd rather we err on the side of allowing the user to decide.

Reply:
if we are testing with go1.12, that's fine but if we stop, we should bump so that user can get a nice message.